pkgs/development/ocaml-modules/ocaml-lsp/jsonrpc.nix

```
@@ -10,10 +10,14 @@
 }:
 
 let params =
-  if lib.versionAtLeast ocaml.version "4.13"
+  if lib.versionAtLeast ocaml.version "4.14"
   then {
-    version = "1.10.3";
-    sha256 = "sha256-o6wQc7Byi5T0vbARF3LAq69/9wMkOZRQ6rcVa/rBUfE=";
+    version = "1.11.1";
```

```suggestion
    version = "1.11.1";
```
Current version is: `1.11.3 `